float64[] joint_angles
float64[] task_measures
---
float64[] task_dynamics